### ISSUE TITLE
Nested import groups (stable in 1.25) break completion

### ISSUE THREAD

**Issue by LPGhatguy** [racer-rust/racer#951]
Given:
```rust
use std::collections::HashMap;

HashMap:: // completion works here
```

But:
```rust
use std::{
    collections::HashMap,
};

HashMap:: // completion does not work here
```

I noticed this using the VS Code Rust plugin today.

**Comment by kngwyu** [comment]
I think it should work since #854.
What's your racer's(or rls's) version?

**Comment by LPGhatguy** [comment]
I'm using whatever the VS Code extension installs, [since their setup guide tells you to skip explicitly installing RLS yourself](https://github.com/rust-lang-nursery/rls#step-1-install-rustup).

I suspect that since I'm on stable and not nightly, the version is before the PR you linked. I'll switch the project over to nightly and see what happens.

**Comment by LPGhatguy** [comment]
Agh. Completion got worse switching to nightly.

Here I'm expecting to see the members of `HashMap`, but I just get `HashMap`:

![image](https://user-images.githubusercontent.com/654598/45243163-98583400-b2a7-11e8-8bb3-43bf36e47a12.png)

Here I'm expecting to see all of the members of `HashMap` that accept self as a first parameter, but I just get... a bunch of unrelated things starting with the letter F?

![image](https://user-images.githubusercontent.com/654598/45243143-8d050880-b2a7-11e8-9e87-07d037140296.png)

**Comment by kngwyu** [comment]
Please let me know the output of
```
rls --version
``:

**Comment by kngwyu** [comment]
I can't reproduce.
If you still has any problem, please add `"rust-client.logToFile": true` to vs-code setting and open it with `env RUST_LOG=rls=debug` and upload the log file. Sorry to have trouble :sweat:

**Comment by cfsamson** [comment]
I have the same problem as well. Code completions basically doesn't work outside of src/main.rs file. Have the same problem across 3 windows machines but not on my Mac. 

Did you figure this out?

**Comment by kngwyu** [comment]
> across 3 windows machines but not on my Mac

Sounds weird...
Thanks for reporting.

**Comment by cfsamson** [comment]
Yeah, it's really annoying. I should mention though I use the Rust RLS plugin in VS Code, but as I understand the code completion is still from Racer. Works fairly well in main.rs but outside that file I have the exact same problem as described above, like this:

![racer](https://user-images.githubusercontent.com/8337848/46889214-3456fb80-ce63-11e8-9a3d-f9272fb3cbc3.png)

Here the objects/traits/methods from the tiberius crate should be listed. Same happens if  write std:
![racer2](https://user-images.githubusercontent.com/8337848/46889350-b515f780-ce63-11e8-92a4-8e8638aa9b51.png)

But if I write std:: in main.rs it lists as expected:
![racer3](https://user-images.githubusercontent.com/8337848/46889365-c101b980-ce63-11e8-8b35-1b094e38ccb4.png)
